plugins/sqlfluff-templater-dbt/test/templater_test.py

```diff
@@ -1,27 +1,25 @@
 """Tests for the dbt templater."""
 
-from copy import deepcopy
-import json
 import glob
-import os
+import json
 import logging
+import os
 import shutil
+from copy import deepcopy
 from pathlib import Path
+from test.fixtures.dbt.templater import (  # noqa: F401
+    DBT_FLUFF_CONFIG,
+    dbt_templater,
+    project_dir,
+)
 from unittest import mock
 
 import pytest
-
+from sqlfluff.cli.commands import lint
```

Comment:
`ruff` is grouping `pytest` and `sqlfluff` together here, which I think is also because first part library detection is missing `sqlfluff`.